src/clue.js

```diff
@@ -1,26 +1,154 @@
+image: "http://www.radiotimes.com/uploads/images/Original/111967.jpg",
+color: "white"
+},
+
+{
+firstName: "Victor",
+lastName: "Plum",
+occupation: "Designer",
+age: 22,
+description: "Billionaire video game designer",
+image: "https://66.media.tumblr.com/ee7155882178f73b3781603f0908617c/tumblr_phhxc7EhPJ1w5fh03_540.jpg",
+color: "purple"
+},
+
+{firstName: "Kasandra",
+lastName: "Scarlet",
+occupation: "Actor",
+age: 31,
+description: "She is an A-list movie star with a dark past",
+image: "https://www.radiotimes.com/uploads/images/Original/111967.jpg",
+color: "red"
+},
+
+{
+firstName: "Eleanor",
+lastName: "Peacock",
+occupation: "Socialité",
+age: 36,
+description: "She is from a wealthy family and uses her status and money to earn popularity",
+image: "https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg",
+color: "blue",
+},
+
+{
+firstName: "Jack",
+lastName: "Mustard",
+occupation: "Retired Football player",
+age: 62,
+description: "He is a former football player who tries to get by on his former glory",
+image: "https://static.independent.co.uk/s3fs-public/thumbnails/image/2016/07/04/08/unspecified-3.jpg",
+color: "yellow",
+},
+
+];
+
+
+
 
 // Rooms Array
 
-const roomsArray = [];
+const roomsArray = [
+
+{name: "Dining_Room"},
+{name: "Conservatory"},
+{name: "Kitchen"},
+{name: "Study"},
+{name: "Library"},
+{name: "Billiard_Room"},
+{name: "Lounge"},
+{name: "Ballroom"},
+{name: "Hall"},
+{name: "Spa"},
+{name: "Living_Room"},
+{name: "Observatory"},
+{name: "Theater"},
+{name: "Guest_House"},
+{name: "Patio"},
+]; 
 
 // Weapons Array
 
-const weaponsArray = [];
+const weaponsArray = [
+
+    {name: "rope", weight: 10},
+    {name: "knife", weight: 8},
+    {name: "candlestick", weight: 2},
+    {name: "dumbbell", weight: 30},
+    {name: "poison", weight: 2},
+    {name: "axe", weight: 15},
+    {name: "bat", weight: 13},
+    {name: "trophy", weight: 25},
+    {name: "pistol", weight: 20},
+
+
+];
+
+
+
+
+
+
+
+
 
 
 // ITERATION 2
 
-function selectRandom() {}
 
-function pickMystery() {}
+// At the beginning of the game, players shuffle each of the card stacks to create a combination of _suspect_, _weapon_ and _room_. This will be the mystery to solve.
+
+function selectRandom(arr) {
+    return arr[Math.floor(Math.random() * arr.length)];  // golden 
+}
+
+// console.log(selectRandom(roomsArray));
+
+
+function pickMystery() {
+    let newVariable = {
+        suspect: " ",
+        weapon: " ",
+        room:  " ",
+    }
+    newVariable.suspect =  selectRandom(suspectsArray);
+    newVariable.weapon =  selectRandom(weaponsArray);
+    newVariable.room =  selectRandom(roomsArray);
+
+    return  newVariable;
+};
 
 
 // ITERATION 3
 
-function revealMystery() {}
+function revealMystery(discover) {
+
+return `${discover.suspect.firstName} ${discover.suspect.lastName} killed Mr. Boddy using the ${discover.weapon.name} in the ${discover.room.name}!`
+
```

Comment:
Here, you have everything correct, however, it is good practice if you destructure your `discover` argument.
By destructuring, it clearly expresses which properties you're using within the function.
By extracting the properties into individual variables (suspect, weapon, and room), you can directly reference them in your return statement without needing to use `discover.suspect`, `discover.weapon`, and `discover.room` every time. This simplifies the code and improves readability.
It would look something like this:

```
function revealMystery(discover) {
  let { suspect, weapon, room } = discover;

  return `${suspect.firstName} ${suspect.lastName} killed Mr. Boddy using the ${weapon.name} in the ${room.name}!`;
}
```